PatreonRSS.php

```diff
@@ -199,18 +224,38 @@ protected function printRssItem($item)
     protected function printRssChannelInfo($campaign, $user)
     {
         echo '<title>';
-        echo htmlspecialchars($campaign['creation_name'] . ' Patreon Posts');
+        echo htmlspecialchars($user['full_name'] . '\'s Patreon Posts');
         echo '</title>';
         echo '<description>';
-        echo htmlspecialchars(strip_tags($campaign['summary']));
+        if(isset($campaign['creation_name']))
+        {
+            echo htmlspecialchars(strip_tags($campaign['creation_name']));
+            echo "\n".htmlspecialchars("<hr>")."\n";
+        }
+        if(isset($campaign['summary']))
+            echo htmlspecialchars(strip_tags($campaign['summary']));
```

Comment:
I really don't like multi-line conditionals without parentheses.  Either have the conditional on a single line or add brackets.